lib/mogrify.ex

```diff
@@ -59,12 +59,12 @@ defmodule Mogrify do
   end
 
   defp arguments_for_saving(image, path) do
-    base_arguments = ~w(-write #{path} #{String.replace(image.path, " ", "\\ ")})
+    base_arguments = ~w(-write) ++ [path] ++ [image.path]
```

Comment:
Sure thing. I do admit seeing that did feel weird leaving that one there like that, I'll fix that and push up a change.